sfputil/main.py

```diff
@@ -5,118 +5,386 @@
 # Command-line utility for interacting with SFP transceivers within SONiC
 #
 
-try:
-    import imp
-    import os
-    import sys
+import operator
+import os
+import sys
 
-    import click
-    from sonic_py_common import device_info, logger, multi_asic
-    from tabulate import tabulate
-except ImportError as e:
-    raise ImportError("%s - required module not found" % str(e))
+import click
+import sonic_platform
```

Comment:
@jleveque - We tested this file on a multi-asic and single asic box, except we had to change this line to
<pre>
import sonic_platform.platform
</pre>

In common code, we are seeing a lot of places where we have to make the same change as above for python3 to work. Is this the best way to fix this?

Reply:
I have heard other accounts of this. Why do you need to change it? What is the error you encounter with it like this?

Reply:
admin@Linecard2:$ /usr/bin/python3
Python 3.7.3 (default, Jul 25 2020, 13:03:44) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sonic_platform
>>> platform_chassis = sonic_platform.platform.Platform().get_chassis()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'sonic_platform' has no attribute 'platform'
>>>

Reply:
I don't see this issue on Arista, Dell or Mellanox platforms. Can you see if there is a noticeable difference in the way your sonic-platform packages are structured?

Also, I just pushed a couple new commits. Can you please test the latest changes?

Reply:
@jleveque sorry for the delay.  There was an issue in the packaging of our sonic-platform that we have now resolved and thus don't see the backtrace with the original contents.

Also, I have tested the latest main.py on multi-asic and single asic pizza box, and it works fine.

Reply:
Great to hear! And thanks for testing the changes!